src/Simplic.Icon.UI/View/IconEditor.xaml.cs

```diff
@@ -57,18 +58,40 @@ private void AddRibbonUserControls()
         {
             var group = new RadRibbonGroup
             {
-                Header = "New Icon"
+                Header = "Icon Einstellung"
             };
 
             var btn = new RadRibbonButton
             {
                 CollapseToSmall = CollapseThreshold.WhenGroupIsMedium,
                 LargeImage = iconService.GetByNameAsImage("action_add_16xLG"),
                 Size = Telerik.Windows.Controls.RibbonView.ButtonSize.Large,
-                Text = "Add new Icon"
+                Text = "Hochladen",
+                Name = "addNewIcon"
             };
 
             btn.SetBinding(RadRibbonButton.CommandProperty, new Binding("AddNewIconCommand"));
+            btn.Click += (sender, e) =>
+            {
+                scrollListViewer.ScrollIntoView(scrollListViewer.ItemContainerGenerator.Items[scrollListViewer.ItemContainerGenerator.Items.Count - 1]);
+            };
+            group.Items.Add(btn);
+
+            btn = new RadRibbonButton
+            {
+                CollapseToSmall = CollapseThreshold.WhenGroupIsMedium,
+                LargeImage = iconService.GetByNameAsImage("delete_32x"),
+                Size = Telerik.Windows.Controls.RibbonView.ButtonSize.Large,
+                Text = "Löschen",
+                IsHitTestVisible = false
+            };
+
+            btn.SetBinding(RadRibbonButton.IsHitTestVisibleProperty, new Binding("IsSelectedIcon")
+            {
+                Converter = new VisibilityToBooleanConverter()
+            });
```

Comment:
is that really the the property you should use for that case?

Reply:
I'm not sure what you mean.